Tools/Pipeline/Windows/Controls/FilterOutputControl.cs

```diff
@@ -0,0 +1,138 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.IO;
+using System.Windows.Forms;
+
+namespace MonoGame.Tools.Pipeline.Windows.Controls
+{
+    public partial class FilterOutputControl : TreeView
+    {
+        const int WM_VSCROLL = 0x0115;
+        const int SB_BOTTOM  = 0x07;
+        [System.Runtime.InteropServices.DllImport("user32.dll", CharSet = System.Runtime.InteropServices.CharSet.Auto, SetLastError = false)]
+        static extern IntPtr SendMessage(IntPtr hWnd, uint Msg, int wParam, int lParam);
+
+        BuildIcons _buildIcons;
+        TreeNode tmpIter;
+        
+        Uri folderUri;
+        Uri outputUri;
+
+        public FilterOutputControl(): base()
+        {
+            this._buildIcons = new BuildIcons();
+            this.ImageList = _buildIcons.Icons;
+        }
+        
+        
+        internal void SetBaseFolder(IController controller)
+        {
+            string pl = ((PipelineController)controller).ProjectLocation;
+            if (!pl.EndsWith(System.IO.Path.DirectorySeparatorChar.ToString()))
+                pl += System.IO.Path.DirectorySeparatorChar;
+            folderUri = new Uri(pl);
+
+            string pod = ((PipelineController)controller).ProjectOutputDir;
+            if (!pod.EndsWith(System.IO.Path.DirectorySeparatorChar.ToString()))
+                pod += System.IO.Path.DirectorySeparatorChar;
+            pod = Path.Combine(pl, pod);
+            outputUri = new Uri(pod);
+        }
+
+
+        internal void AppendText(string text)
```

Comment:
Yes. I did like file.exists either. I was looking for a .IsPath/.IsFilename type if method but the closest was .IsRooted which means that using exceptions for code flow. I thought that since the name comes ftom mgcb then I would get an error before the file but really never tested it.
I guess what I need here is a Regular expression to test if the line is a file path and just that.